.github/workflows/wcag_test.yml

```diff
@@ -0,0 +1,19 @@
+name: WCAG tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  call_wcag_test:
+    uses: yext/slapshot-reusable-workflows/.github/workflows/wcag_test.yml@v1
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    with:
+      build_script: |
+        npm run storybook -- --quiet &
```

Comment:
can we use the same approach as the one for visual-coverage script? with max time out and terminate if storybook is not served?